Blazor_Foundation_6_Test_Shared/Documentation/Containers/Callout.razor

```diff
@@ -1,6 +1,119 @@
+            <h5>This is an alert callout.</h5>
+            <p>It has an easy to override visual style, and is appropriately subdued.</p>
+            <NavLink href="#">It's dangerous to go alone, take this.</NavLink>
+        </OpenCodeDev.Blazor.Foundation.Components.Containers.Callout>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Sizing</h2>
+        <p>Callouts can be sized using Size <Code>Size="small"</Code> and <Code>Size="large"</Code>. These will affect the padding around content to be smaller and larger respectively.</p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PENhbGxvdXQgU2l6ZT0ic21hbGwiPgogICAgPGg1PlRoaXMgaXMgYSBzbWFsbCBjYWxsb3V0PC9oNT4KICAgIDxwPkl0IGhhcyBhbiBlYXN5IHRvIG92ZXJyaWRlIHZpc3VhbCBzdHlsZSwgYW5kIGlzIGFwcHJvcHJpYXRlbHkgc3ViZHVlZC48L3A+CiAgICA8TmF2TGluayBocmVmPSIjIj5JdCdzIGRhbmdlcm91cyB0byBnbyBhbG9uZSwgdGFrZSB0aGlzLjwvTmF2TGluaz4KPC9DYWxsb3V0PgoKPENhbGxvdXQgU2l6ZT0ibGFyZ2UiPgogICAgPGg1PlRoaXMgaXMgYSBsYXJnZSBjYWxsb3V0PC9oNT4KICAgIDxwPkl0IGhhcyBhbiBlYXN5IHRvIG92ZXJyaWRlIHZpc3VhbCBzdHlsZSwgYW5kIGlzIGFwcHJvcHJpYXRlbHkgc3ViZHVlZC48L3A+CiAgICA8TmF2TGluayBocmVmPSIjIj5JdCdzIGRhbmdlcm91cyB0byBnbyBhbG9uZSwgdGFrZSB0aGlzLjwvTmF2TGluaz4KPC9DYWxsb3V0Pg==" />
+
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.Callout Size="small">
+            <h5>This is a small callout</h5>
+            <p>It has an easy to override visual style, and is appropriately subdued.</p>
+            <NavLink href="#">It's dangerous to go alone, take this.</NavLink>
+        </OpenCodeDev.Blazor.Foundation.Components.Containers.Callout>
+
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.Callout Size="large">
+            <h5>This is a large callout</h5>
+            <p>It has an easy to override visual style, and is appropriately subdued.</p>
+            <NavLink href="#">It's dangerous to go alone, take this.</NavLink>
+        </OpenCodeDev.Blazor.Foundation.Components.Containers.Callout>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+
+
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Making Closable</h2>
+        <p>Pair the callout with the close button component and <Code>ClosableData</Code> attribute to create a dismissable alert box.</p>
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.Callout TypeClass="primary">
+            Any element can be used as a close trigger, not just close button. Adding the attribute <Code>CloseButton="true"</Code> to any element within the callout will turn it into a close trigger.
+
+            When using the <Code>ClosableData</Code> attribute, you can optionally add <a href="https://get.foundation/sites/docs/motion-ui.html">Motion UI</a> classes to the attribute to change the closing animation. If no class is added, the plugin defaults to jQuery's <Code>.fadeOut()</Code>  function.
```

Comment:
Reformulate this entire statement. Well, the reference to original Foundation doc is good but JQuery, Class and JS FUNCTION !!!!! stuff doesn't make any senses for Blazor! you should know that.
you should remove the entire block and link the foundation website like you did but making it closable should focus on close button. Make a sub header to talk about closing animation with some exemple and dont make any promises like Any element can be used as a close trigger